src/Components/Onboarding/Wizard.tsx

```diff
@@ -61,17 +65,24 @@ class Wizard extends React.Component<Props, State> {
     return <CurrentStep onNextButtonPressed={this.onNextButtonPressed} />
   }
 
+  onFinish = () => {
+    const { redirectTo } = this.props
+    this.setState({ percentComplete: 1 })
+    setTimeout(() => {
+      window.location.href = redirectTo
+    }, 500)
+
+    this.props.tracking.trackEvent({
+      action: "Completed Onboarding",
+    })
+  }
```

Comment:
This is the only track call that will be lost when we remove the Wizard but I can follow up when we transfer to the react-router implementation.